### ISSUE TITLE
Power management options for BMI270 driver

### ISSUE THREAD

**Issue by Kampi** [jakkra/ZSWatch#154]
**ToDo**

- [ ] Proove that the sensor is working when power saving options are enabled
- [ ] Add driver code for power management

**Comment by jakkra** [comment]
Low prio, as it's already consuming quite little and as long as rest of the system is >> BMI270, getting other parts of ZSWatch lower power is higher prio.

**Comment by Kampi** [comment]
> Low prio, as it's already consuming quite little and as long as rest of the system is >> BMI270, getting other parts of ZSWatch lower power is higher prio.

Yes. Just to get it out of my head and as a collection list for features :)
